### PR TITLE
Allow adding of names to countries for office sweepstake

### PR DESCRIPTION
**Pull request by rutson**
Options 'players' array added to slack_instances.

### CODE REVIEW

**Comment by justcallmelarry** [comment]
No problem adding this, as it would not do anything for the people not using the function.
However the name "players" kinda feels weird, as I would assume that means the players of the team, not the office sweepstake.

Maybe "participant" or similar?

File: google.py [hunk 1]
```diff
@@ -243,6 +243,10 @@ def status(self, text, match, conlist):
                 self.logger.error(e)
         for si in self.slack_instances:
             output = dict(self.slack_payload)
+            if si.get('players'):
+                for country,name in si.get('players').items():
+                    newtext = "%s (%s)"%(country,name)
```
Comment (justcallmelarry) [inline google.py]: should be single quoted and using the same format strings as the rest of the repo (ie: f'{country} {name}'

**Review by justcallmelarry — CHANGES_REQUESTED** [review]

**Comment by rutson** [comment]
Yes, that makes perfect sense - 'players' was borrowed from aakarim/slack-world-cup-sweepstakes - I'll change to participants tonight.

File: google.py (hunk 1 above)
Comment (justcallmelarry) [reply google.py]: and "country,name" => "country, name" (add space) to follow flake8 standard

**Pull request merged**